stage1/init/common/pod.go

```diff
@@ -1098,3 +1063,44 @@ func getAppNoNewPrivileges(isolators types.Isolators) bool {
 
 	return false
 }
+
+// restrictProcFS restricts access to some security-sensitive paths under
+// /proc and /sys. Entries are either hidden or just made read-only to app.
+func restrictProcFS(opts []*unit.UnitOption, appName types.ACName) []*unit.UnitOption {
```

Comment:
Since it is not only files in `/proc`, the function could be called `protectSystemFiles`. Similar terminology as [ProtectSystem in systemd](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectSystem=)..